build.sbt

```diff
@@ -12,7 +12,8 @@ lazy val root = (project in file(".")).settings(
   name := "LibreCaptcha",
   libraryDependencies += "com.sksamuel.scrimage" % "scrimage-core" % "4.0.12",
   libraryDependencies += "com.sksamuel.scrimage" % "scrimage-filters" % "4.0.12",
-  libraryDependencies += "org.json4s" % "json4s-jackson_2.13" % "3.6.11"
+  libraryDependencies += "org.json4s" % "json4s-jackson_2.13" % "3.6.11",
+  libraryDependencies += "com.sun.net.httpserver" % "http" % "20070405"
```

Comment:
Is this needed. On my machine it is available by default!

Reply:
It was throwing an error in the beginning. Maybe has something to do with how packages are discovered/imported?

Reply:
I have removed it since it's working fine without declaring it explicitly.